www/__mocks__/cordovaMocks.ts

```diff
@@ -87,6 +87,33 @@ export const mockBEMUserCache = () => {
           rs(messages.filter(m => m.key == key).map(m => m.value));
         }, 100)
       );
+    },
+    getDocument: (key: string, withMetadata?: boolean) => {
+      // this was mocked specifically for enketoHelper's use, could be expanded if needed
+      const fakeSurveyConfig = {
+        survey_info: {
+          surveys: {
+            TimeUseSurvey: { compatibleWith: 1, 
+              formPath: "https://raw.githubusercontent.com/sebastianbarry/nrel-openpath-deploy-configs/surveys-info-and-surveys-data/survey-resources/data-json/time-use-survey-form-v9.json", 
+              labelTemplate: {en: "{ erea, plural, =0 {} other {# Employment/Education, } }{ da, plural, =0 {} other {# Domestic, } }",
+                              es: "{ erea, plural, =0 {} other {# Empleo/Educación, } }{ da, plural, =0 {} other {# Actividades domesticas, }}"}, 
+              labelVars: {da: {key: "Domestic_activities", type: "length"},
+                          erea: {key: "Employment_related_a_Education_activities", type:"length"}}, 
+              version: 9}
+          }
+        }
+      }
```

Comment:
I do think it would be nice to extract this 'fake' config to a separate file, and also include more things in it besides just `survey_info`. Like you said, we'll likely want to expand on it later.

Reply:
Ok, I'll work on doing that. I'm a little worried about setting it up in a way that doesn't work with the next application that tries to use it, but we can always update. I got this section by reverse-engineering from log statements, but can base the rest on one of the `nrel-deploy-configs`.

Reply:
I moved this into it's own file and added the rest of the fields in [b480279](https://github.com/e-mission/e-mission-phone/pull/1063/commits/b48027958eb25a1e2f79f75dec53abd3f93a2ab8). I placed the new file in the `__mocks__` folder, but if it would be better somewhere else I can move it.